LoRaEngine/test/LoRaWan.IntegrationTest/ABPTest.cs

```diff
@@ -220,6 +220,81 @@ public async Task Test_ABP_Invalid_NwkSKey_Fails_With_Mic_Error()
 
             // Before starting new test, wait until Lora drivers stops sending/receiving data
             await this.ArduinoDevice.WaitForIdleAsync();
-        }        
+        }
+
+        // Verifies that ABP confirmed and unconfirmed messages are working
+        // Uses Device5_ABP
+        [Fact]
+        public async Task Test_ABP_Device_With_Same_DevAddr()
+        {
+
+
+            const int MESSAGES_COUNT = 2;
+            Log($"[INFO] ** Starting {nameof(Test_ABP_Confirmed_And_Unconfirmed_Message)} using devices {this.TestFixture.Device16_ABP.DeviceID} and {this.TestFixture.Device17_ABP.DeviceID} **");
```

Comment:
Correct the name used in log `nameof(Test_ABP_Device_With_Same_DevAddr)`

Reply:
Sorry my bad, forgot this one, It is corrected now